frontend/src/APIClients/CheckInAPIClient.ts

```diff
@@ -0,0 +1,110 @@
+import { BEARER_TOKEN } from "../constants/AuthConstants";
+import { CheckIn } from "../types/CheckInTypes";
+import baseAPIClient from "./BaseAPIClient";
+
+const getAllCheckIns = async (): Promise<CheckIn[]> => {
+  try {
+    const { data } = await baseAPIClient.get("/checkin", {
+      headers: { Authorization: BEARER_TOKEN },
+    });
+    return data;
+  } catch (error) {
+    return [];
```

Comment:
should this be `return error as CheckIn[]`

Reply:
Yep. I originally based this off of the SchedulingAPIClient.ts and they return [] instead of return `error as Schedule[]`. But looking at the other APIClients, they all return `error as Donor[]/VolunteerResponse[]`, etc. Made the change.